### PR TITLE
Remove styling from AudioPlayer component

### LINKED ISSUE #6877: Move AudioPlayer styling into OD Radio + Live Radio page components
**Is your feature request related to a problem? Please describe.**

We should move the styling from the audio player into the parent page components to render the audio player component reusable in other media page types.

**Describe the solution you'd like**
The audio player component should not contain any styling.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by rebeccamcginn**
Resolves #6877

**Overall change:**
Moved styling out of AudioPlayer and into ODRadioPage + LiveRadioPage components

**Code changes:**

- Moved styling from AudioPlayer to OD Radio and Live Radio page components
- Moved 'expired content' message from AudioPlayer to OD Radio page component

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/LiveRadioPage/index.jsx
Comment (karinathomasbbc) [inline src/app/pages/LiveRadioPage/index.jsx]: Would `StyledAudioPlayer` be a more consistent naming convention?

File: src/app/pages/OnDemandRadioPage/index.jsx
Comment (karinathomasbbc) [inline src/app/pages/OnDemandRadioPage/index.jsx]: Would `StyledMediaMessage` be a more consistent naming convention?

**Review by ryanmccombe — APPROVED** [review]
👍

File: src/app/pages/OnDemandRadioPage/index.jsx
Comment (karinathomasbbc) [inline src/app/pages/OnDemandRadioPage/index.jsx]: Would `StyledAudioPlayer` be more consistent here?

Or, we could do something like this:
```suggestion
// I don't know if this actually works
import _AudioPlayer  from '#containers/AudioPlayer';
// If it doesn't work try this instead?
// import { AudioPlayer as _AudioPlayer } from '#containers/AudioPlayer';
// Last resort
// import { default as _AudioPlayer } from '#containers/AudioPlayer';
...
const AudioPlayer = styled(_AudioPlayer)`
```

File: src/app/pages/OnDemandRadioPage/index.jsx
Comment (ryanmccombe) [inline src/app/pages/OnDemandRadioPage/index.jsx:163]: Not for this, but something for the tech debt / make tiny changes backlog might be to remove the `position: absolute` from this `MediaMessage` component

That would let us remove these `MediaMessageWrapper`s

File: src/app/pages/OnDemandRadioPage/index.jsx
Comment (rebeccamcginn) [reply src/app/pages/OnDemandRadioPage/index.jsx]: Please can you explain what the underscore means here?

Comment (karinathomasbbc) [reply src/app/pages/OnDemandRadioPage/index.jsx]: I've seen an example of this used here and thought it was neat:
https://github.com/bbc/simorgh/blob/latest/src/app/pages/index.js#L39

But `StyledAudioPlayer` would also work.

**Review by karinathomasbbc — APPROVED** [review]

**Comment by rebeccamcginn** [comment]
Please do not merge until Lily has signed off on the work

**Pull request merged**
